### ISSUE TITLE
Game server request - SoulMask

### ISSUE THREAD

**Issue by bobby86928** [DiscordGSM/GameServerMonitor#98]
hope can support this game
https://store.steampowered.com/app/2646460/Soulmask/

I am using NO-IP, so the domain name is `taiwansecurity.ddns.net`
port is `27015`

thanks a lot

**Comment by xLeviNx** [comment]
Works fine on self host. Waiting for battlefields merge

**Comment by darkcloud784** [comment]
I dont see this on the listed supported games, has it been added yet?

**Comment by xLeviNx** [comment]
It has on self host. I am adding some changes soon.

**Comment by darkcloud784** [comment]
> It has on self host. I am adding some changes soon.

I've tried adding my soulmask server but it says its an invalid game.

soulmask is not a valid game id. Learn more: https://discordgsm.com/guide/supported-games

**Comment by xLeviNx** [comment]
https://github.com/xLeviNx/GameServerMonitor

Follow the steps to install on docker but use this link instead.

**Comment by darkcloud784** [comment]
> https://github.com/xLeviNx/GameServerMonitor
> 
> Follow the steps to install on docker but use this link instead.

Is there no already built images I can pull? I use docker compose and portainer so its a pain to have to build the image on every update.

**Comment by xLeviNx** [comment]
Ah not sure
https://github.com/xLeviNx/GameServerMonitor/releases/tag/2.16.3
I put a release so you could use that. I think its just changing two lines that should automatically pull it?

**Comment by darkcloud784** [comment]
> Ah not sure https://github.com/xLeviNx/GameServerMonitor/releases/tag/2.16.3 I put a release so you could use that. I think its just changing two lines that should automatically pull it?

No, this just releases shows the code and allows you to download it as a zip or tar.gz. You'd have to create a workflow for building a docker image and then release that as a package.

https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-docker-images

**Comment by Necrul** [comment]
> > Ah not sure https://github.com/xLeviNx/GameServerMonitor/releases/tag/2.16.3 I put a release so you could use that. I think its just changing two lines that should automatically pull it?
> 
> No, this just releases shows the code and allows you to download it as a zip or tar.gz. You'd have to create a workflow for building a docker image and then release that as a package.
> 
> https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-docker-images

You ever get Soulmask working somehow? It's on the list of supported games but as you got...mine says invalid if I try to add it.

**Comment by swesner411** [comment]
This was resolved in #99 and can probably be closed.
